haproxy.cfg

```diff
@@ -28,21 +28,20 @@ defaults
     load-server-state-from-file global
 
     # Use provided example error pages
-    errorfile 400 /usr/local/etc/haproxy/errors/400.http
-    errorfile 403 /usr/local/etc/haproxy/errors/403.http
-    errorfile 408 /usr/local/etc/haproxy/errors/408.http
-    errorfile 500 /usr/local/etc/haproxy/errors/500.http
-    errorfile 502 /usr/local/etc/haproxy/errors/502.http
-    errorfile 503 /usr/local/etc/haproxy/errors/503.http
-    errorfile 504 /usr/local/etc/haproxy/errors/504.http
+    # errorfile 400 /usr/local/etc/haproxy/errors/400.http
+    # errorfile 403 /usr/local/etc/haproxy/errors/403.http
+    # errorfile 408 /usr/local/etc/haproxy/errors/408.http
+    # errorfile 500 /usr/local/etc/haproxy/errors/500.http
+    # errorfile 502 /usr/local/etc/haproxy/errors/502.http
+    # errorfile 503 /usr/local/etc/haproxy/errors/503.http
+    # errorfile 504 /usr/local/etc/haproxy/errors/504.http
```

Comment:
Why this change? :thinking:

Reply:
Not sure, maybe by accident. I'll revert this :)